doc/sphinx/source/iio_osc/data_streaming.rst

```diff
@@ -0,0 +1,127 @@
+===========================
+IIO Data Streaming for DACs
+===========================
+
+Introduction
+============
+
+IIO clients can be used to load and stream data to DAC devices using an IIO 
+firmware application developed for that device. Using this firmware, a user 
+can configure various settings on the device and load the data using the DAC 
+data manager tab. The data is transmitted using Virtual Serial or UART serial 
+link. The input data to be streamed can be loaded in a binary data file format. 
+
+The channels for which data is to be streamed can be selected from the GUI window, 
+and the number of samples per channel will be taken from the input data file provided. 
+So, for an 8-channel DAC, if all 8 channels are selected for streaming, the input 
+file should contain 3200 samples, considering 400 samples per channel.
+
+IIO client allows data to be streamed to the iio device in two ways: either cyclic 
+streaming enabled or disabled. When the cyclic buffer is 
+enabled (cyclic buffers are identical or normal or non-cyclic buffers, except when 
```

Comment:
nit: the part in the parentheses is confusing - I'm not sure I follow what's been conveyed here

Reply:
I see. I think there's a typo then: 'identical **or** normal' --> 'identical **to** normal'

I agree, removing that part (and leaving the rest of the sentences unchanged) will make the explanation simpler and more concise